patterns/product-details-pattern.php

```diff
@@ -0,0 +1,136 @@
+		<figure class="wp-block-image size-full is-resized">
+			<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/product-furniture-3.png', dirname( __FILE__ ) ) ); ?>" alt="" class="wp-image-1" width="500" height="723"/>
+		</figure>
+		<!-- /wp:image -->
+
+		<!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"0","left":"0"},"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"margin":{"top":"0.2em","bottom":"0.2em"}}}} -->
+		<div class="wp-block-columns" style="margin-top:0.2em;margin-bottom:0.2em;padding-top:0;padding-right:0;padding-bottom:0;padding-left:0">
+			<!-- wp:column {"width":"243px","style":{"spacing":{"padding":{"top":"0px","right":"5px","bottom":"0px","left":"0px"},"blockGap":"0"}},"layout":{"type":"default"}} -->
+			<div class="wp-block-column" style="padding-top:0px;padding-right:5px;padding-bottom:0px;padding-left:0px;flex-basis:243px">
+				<!-- wp:image {"id":1,"sizeSlug":"full","linkDestination":"none"} -->
+				<figure class="wp-block-image size-full">
+					<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/product-furniture-2.png', dirname( __FILE__ ) ) ); ?>" alt="" class="wp-image-1"/>
+				</figure>
+				<!-- /wp:image --></div>
+			<!-- /wp:column -->
+
+			<!-- wp:column {"width":"244px","style":{"spacing":{"padding":{"top":"0","right":"0","bottom":"0","left":"0"},"blockGap":"0px"}},"layout":{"type":"default"}} -->
+			<div class="wp-block-column" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0;flex-basis:244px">
+				<!-- wp:image {"id":1,"sizeSlug":"full","linkDestination":"none"} -->
+				<figure class="wp-block-image size-full">
+					<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/product-furniture-1.png', dirname( __FILE__ ) ) ); ?>" alt="" class="wp-image-1"/>
+				</figure>
+				<!-- /wp:image --></div>
+			<!-- /wp:column --></div>
+		<!-- /wp:columns --></div>
+	<!-- /wp:column -->
+
+	<!-- wp:column -->
+	<div class="wp-block-column">
+		<!-- wp:heading {"textColor":"foreground","fontSize":"x-large"} -->
+		<h2 class="wp-block-heading has-foreground-color has-text-color has-x-large-font-size">Herriott 40" Solid Wood Console Table</h2>
+		<!-- /wp:heading -->
+
+		<!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px","margin":{"top":"10px","bottom":"0px"}}},"layout":{"type":"flex","flexWrap":"nowrap"}} -->
+		<div class="wp-block-group" style="margin-top:10px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px">
+			<!-- wp:image {"id":1,"width":90,"height":20,"sizeSlug":"full","linkDestination":"none","style":{"layout":{"selfStretch":"fixed","flexSize":"90px"}}} -->
+			<figure class="wp-block-image size-full is-resized">
+				<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/rating-stars.png', dirname( __FILE__ ) ) ); ?>" alt="" class="wp-image-1" width="90" height="20"/>
```

Comment:
What do you think about changing the row containing the stars and ratings by something like this? 🤔 
```<!-- wp:group {"style":{"spacing":{"padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"blockGap":"0px","margin":{"top":"10px","bottom":"0px"}}},"layout":{"type":"flex","flexWrap":"nowrap","verticalAlignment":"center"}} -->
<div class="wp-block-group" style="margin-top:10px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:paragraph {"textColor":"luminous-vivid-amber"} -->
<p class="has-luminous-vivid-amber-color has-text-color">★★★★</p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"style":{"color":{"text":"#ffe8a4"},"spacing":{"margin":{"right":"5px"}}}} -->
<p class="has-text-color" style="color:#ffe8a4;margin-right:5px">★</p>
<!-- /wp:paragraph -->

<!-- wp:paragraph {"style":{"typography":{"fontSize":"0.7em"}},"textColor":"foreground"} -->
<p class="has-foreground-color has-text-color" style="font-size:0.7em">  <strong>4.2</strong>(1,079 reviews)</p>
<!-- /wp:paragraph --></div>
<!-- /wp:group -->

This way we can remove the starts image and the ratings alignment look a bit better.